bigchaindb/common/transaction.py

```diff
@@ -274,6 +274,15 @@ def to_dict(self):
                 'output_index': self.output,
             }
 
+    def to_query(self):
+        if self.txid is None and self.output is None:
```

Comment:
Is it possible for just one of these to be `None`?

Reply:
Actually, these fields cannot be `None`. I started off with `to_dict`, this code is leftover